website/events/emails.py

```diff
@@ -25,12 +25,19 @@ def notify_first_waiting(event):
         subject = _("[THALIA] Notification about your registration for '{}'").format(
             event.title
         )
+
```

Comment:
You probably want to `", ".join()` this list, otherwise it will actually be formatted as a python list